autogpt_platform/backend/backend/blocks/llm.py

```diff
@@ -238,6 +242,190 @@ class Message(BlockSchema):
     content: str
 
 
+class LLMResponse(BaseModel):
+    prompt: str
+    response: str
+    tool_calls: Optional[List[ChatCompletionMessageToolCall]] | None
```

Comment:
imo don't use provider-specific library types to define an internal type like this